### PR TITLE
Wireframes read me

### PR DESCRIPTION
**Pull request by mugendiz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ This setup allows for a Node/Express/React app which can be easily deployed to H
...
+### Login Page
+<img src="Wireframe_Images/Podcast-Wireframe3.png" width="400">
+
+### Favorites
+<img src= "Wireframe_Images/Podcast-Wireframe4.png" width="400">
+
+### Subscribed
+<img src= "Wireframe_Images/Podcast-Wireframe5.png" width="400">
+
+
```
Comment (mbychkowski) [inline README.md]: The source for these images might have to change once it is up on GitHub. Make sure you change it appropriatley.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+app.use('/auth', authRoutes);
+
+// create home route
+app.get('/', (req, res) => {
+    res.render('home');
+});
+
+app.listen(3000, () => {
+    console.log('app now listening for requests on port 3000');
+});
```
Comment (mbychkowski) [inline app.js]: What is this `app.js` file for?

Currently this stuff is in `server.js`.

Comment (mbychkowski) [inline app.js]: What is this `app.js` file for?

Currently this stuff is in `server.js`.

File: README.md (hunk 1 above)
Comment (mugendiz) [reply README.md]: i believe github supports this html for the images when done this way

File: config/keys.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// add this file to .gitignore
+
+module.exports = {
+        google: {
+            clientID: '558027906323-30k4oqhfe0m38iupmhef8p1hid76v6i5.apps.googleusercontent.com',
+            clientSecret: 'OzdguWo6g2TaUfrhOe8lPGse'
+        }
+};
```
Comment (mbychkowski) [inline config/keys.js]: I don't think this should be pushed into stage. Update .gitignore appropriately to make sure this isn't being pushed to GitHub.

Will need to be removed from GitHub as well.

File: .gitignore [hunk 4]
```diff
@@ -18,3 +18,5 @@ client/node_modules
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+keys.js
```
Comment (mbychkowski) [inline .gitignore]: I think it needs to be `/client/keys.js`

See comments below. This is not being ignored.

File: README.md (hunk 1 above)
Comment (mbychkowski) [reply README.md]: okay, cool. In the past I have always had to copy the exact URL link from GitHub for the image once it was uploaded.

Comment (mbychkowski) [reply README.md]: Just checked the README on your branch. Cool! I will need to do it that way from now on.

**Pull request closed without merging**